### PR TITLE
Penthrite rework: now it's actually usable![READY]

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request
Penthrite rework - Now it is a medicine simmilair to atropine, but potentially even better. It prevents you from going into hard and soft crit giving you practical additional 60 total hp. The trick is that if you go above that 160 total hp you loose all benefits of penthrite and get a heart attack. It also deals heart damage based on the volume. It's heart stabilizing effect has not been changed, but it's explosion reaction has been. Now it explodes in contact with epinephrine and atropine.

## Why It's Good For The Game

Penthrite was never really used, it had shitty explosion , it was horrible to craft and it was very bad at it's job since it dealt stomach damage. Now it is much better with bigger drawbacks. 

If this is merged we will have 3 emergency chems:

Epinephrine(easy) < Atropine(safe) < Penthrite(Risky)

You make penthrite by mixing in equal amounts : 
- pentaerythritol
- acetone
- nitric acid
- wittel

## Changelog
:cl:
tweak: Penthite now can extend your life span or something. Scientists warned me not to mix it with epinephrine or atropine, noone is allowed to tell me what to not do!
/:cl:

### CODE REVIEW

**Comment by davethwave** [comment]
From the description, you have just created a fearsome poison.

**Comment by EdgeLordExe** [comment]
> From the description, you have just created a fearsome poison.

not really , worst effects kick in late in the exposure and if you have no addictions you will just vomit it away.

**Comment by ArcaneDefence** [comment]
I was thinking more along the lines of using penthrite exclusively to murder full health people with relative ease, or as a possible replacement for RDX/teslium

**Comment by ExcessiveUseOfCobblestone** [comment]
Penthrite's downside is that it hurts the heart but penthrite allows you to function without a heart so...?

I thought I had this conversation with someone yesterday but I told them now that roundstart organs are available via cybernetics, we don't really need  the -rite chems anymore (they don't get used much anyways).

As for addictions, They're kinda not in a good spot rn considering they're a massive PITA to fix (can you even fix them w/o body transfer?) and Addiction is really just a mega OD, which is already near exclusive to antag shenanigans or people who do not know how to use chems in general. I'd rather see addiction be based on cycles or tracked throughout the round on the mob so the focus of addiction is prolonged exposure (which is affected by any amount in the body), vs. having a lot of the reagent in you once.

That said, I'd like to keep the severity of Addiction but make it more difficult to obtain in a short span. It should take a bit to get addiction, and it should impact the rest of your round unless you seriously go out of your way to remedy it. A Chem that can null that (even if it does split damage) will severely hamper that severity, especially if someone just makes the magic aki ito pills.

**Comment by EdgeLordExe** [comment]
> Penthrite's downside is that it hurts the heart but penthrite allows you to function without a heart so...?
> 
> I thought I had this conversation with someone yesterday but I told them now that roundstart organs are available via cybernetics, we don't really need the -rite chems anymore (they don't get used much anyways).
> 
> As for addictions, They're kinda not in a good spot rn considering they're a massive PITA to fix (can you even fix them w/o body transfer?) and Addiction is really just a mega OD, which is already near exclusive to antag shenanigans or people who do not know how to use chems in general. I'd rather see addiction be based on cycles or tracked throughout the round on the mob so the focus of addiction is prolonged exposure (which is affected by any amount in the body), vs. having a lot of the reagent in you once.
> 
> That said, I'd like to keep the severity of Addiction but make it more difficult to obtain in a short span. It should take a bit to get addiction, and it should impact the rest of your round unless you seriously go out of your way to remedy it. A Chem that can null that (even if it does split damage) will severely hamper that severity, especially if someone just makes the magic aki ito pills.

when it comes to penthrite, yeah i agree. thats why i changed the chem to be more like atropine or epinephrine. It is an emergency chem that can literally bring back someone from hard crit. 

last time i checked LDX kills you before you get your first trauma cured, meaning that it requires constant medical attention or the person will just simply die. I can probably make it even harsher so that it is impossible to cure an addiction AND get it without constant medical supervision.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1305,3 +1305,45 @@
 	L.remove_movespeed_mod_immunities(type, /datum/movespeed_modifier/damage_slowdown)
 	L.Dizzy(0)
 	L.Jitter(0)
+
+/datum/reagent/medicine/ldx
+	name = "LDX"
+	description = "A blood red thick liquid patented by NT in it's early years. It's primary purpose is to quickly remove any addctions from the patient, it accomplishes that goal very violently."
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1311]: Commas and order of adjectives. Shortening Nanotrasen to NT is weird. Missing but. Also a typo. Also "it's primary purpose is to" makes the sentence feel drawn out. 50 cycles is roughly 100 seconds, so i wouldn't say quickly. And an unnecessary any.
```suggestion
	description = "A thick, blood red liquid patented by Nanotrasen in it's early years. It slowly removes addictions from the patient, but it accomplishes that goal very violently."
```

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -1305,3 +1305,45 @@
...
+	. = ..()
+	switch(current_cycle) //ramp up effect, the longer it is in our system the more damage it deals
+		if(9)
+			M.Dizzy(pick(0,1,2))
+			M.Jitter(pick(0,1,2))
+			M.adjustToxLoss(0.5)
+
+		if(20 to 39)
+
+			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "pain", /datum/mood_event/ldx_pain, name)
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1328]: Maybe give a message about the pain with a to_chat? Something like "You don't feel good." or "You feel an immense pain in your abdomen." for the later stages

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 3]
```diff
@@ -115,11 +115,15 @@
 	required_temp = 550 + rand(-49,49)
 
 
-/datum/chemical_reaction/reagent_explosion/penthrite_explosion
-	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/phenol = 1, /datum/reagent/acetone_oxide = 1)
-	required_temp = 315
+/datum/chemical_reaction/reagent_explosion/penthrite_explosion1
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm:118]: Magic typepath.
```suggestion
/datum/chemical_reaction/reagent_explosion/penthrite_explosion_epinephrine
```

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 4]
```diff
@@ -115,11 +115,15 @@
...
+	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/medicine/epinephrine = 1)
 	strengthdiv = 5
 
+/datum/chemical_reaction/reagent_explosion/penthrite_explosion2
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm:122]: Magic typepath.
```suggestion
/datum/chemical_reaction/reagent_explosion/penthrite_explosion_atropine
```

**Review by trollbreeder — COMMENTED** [review]
Grammar review. There are suggested changes for each review comment, so it'll be easy to fix. Or you can do your own thing.

**Comment by itseasytosee** [comment]
If I'm understanding this correctly you have turned a drug specifically designed to help stabilize people with unstable hearts (even if it was never used due to its beefy recipe) into a chemical that gives people who are healthy heart attacks and lets them move around until there body ceases to function. the only way I could see this effectively used is in a last stand scenario where you are going to die anyway and want a few extra seconds of life before you die.
Also, I think addictions are fine right now and shouldn't lock anyone up in medbay just so they can be removed.
Tldr; you made Initropidril that only works on healthy people

**Comment by EdgeLordExe** [comment]
> If I'm understanding this correctly you have turned a drug specifically designed to help stabilize people with unstable hearts (even if it was never used due to its beefy recipe) into a chemical that gives people who are healthy heart attacks and lets them move around until there body ceases to function. the only way I could see this effectively used is in a last stand scenario where you are going to die anyway and want a few extra seconds of life before you die.
> Also, I think addictions are fine right now and shouldn't lock anyone up in medbay just so they can be removed.
> Tldr; you made craftable Initropidril that only works on healthy people

heart attack is only caused when total damage is greater than 160. as long as you are below that it is better atropine

**Comment by Pugie** [comment]
> > From the description, you have just created a fearsome poison.
> 
> not really , worst effects kick in late in the exposure and if you have no addictions you will just vomit it away.

Every chem you have ever made is a fearsome poison

**Comment by EdgeLordExe** [comment]
> > > From the description, you have just created a fearsome poison.
> > 
> > 
> > not really , worst effects kick in late in the exposure and if you have no addictions you will just vomit it away.
> 
> Every chem you have ever made is a fearsome poison

im not denying that

**Comment by itseasytosee** [comment]
>heart attack is only caused when total damage is greater than 160 (hard crit) . as long as you are below that it is better atropine

Ok I understand now
I sill feel like its a chem that you would only use as a last stand in nukies or something rather than practical medicinal applications due to the fact that once it wears off you are stuck with a broken heart and in critical condition. I would consider it more of a drug than medicine at this point.

**Comment by Tlaltecuhtli** [comment]
addictions can be easily countered by asking for a bunch of 0.1u pills like 1 u can last you the whole round and more, this just makes one of the last thing that felt interesting and somewhat immersive for medical into another eat x and you are fixed

**Comment by ExcessiveUseOfCobblestone** [comment]
girls be like "this chem is useless" while I try to figure out how to optimize my aki ito pill setup to include a path that outputs large volumes of this chem in place of the atropine I usually add to the pill.

You better believe I'm doping up on this chem every round at high volumes, probably could offer pills to put in people so they don't have to carry a beaker or something. You will not catch me in softcrit or hardcrit again, only dead or alive.

Absolutely bonkers of a medicine.

**Comment by EdgeLordExe** [comment]
> girls be like "this chem is useless" while I try to figure out how to optimize my aki ito pill setup to include a path that outputs large volumes of this chem in place of the atropine I usually add to the pill.
> 
> You better believe I'm doping up on this chem every round at high volumes, probably could offer pills to put in people so they don't have to carry a beaker or something. You will not catch me in softcrit or hardcrit again, only dead or alive.
> 
> Absolutely bonkers of a medicine.

you will have to have a heart of steel to sustain yourself on this med

**Comment by HumanNugget** [comment]
chem that keeps u alive past crit but your heart explodes after that
litteraly berserk potion , like one of those liveleak vids where a drug addict survives 5 shots in the chest and keeps fighting 
sounds awesome

**Comment by EdgeLordExe** [comment]
actually cobby i agree with you on the LDX being shit, it is shit. Once i feel like doing codedocs ill make  something new for medical that will have LDX like procedure in it. Now's not the time

**Comment by EdgeLordExe** [comment]
> girls be like "this chem is useless" while I try to figure out how to optimize my aki ito pill setup to include a path that outputs large volumes of this chem in place of the atropine I usually add to the pill.
> 
> You better believe I'm doping up on this chem every round at high volumes, probably could offer pills to put in people so they don't have to carry a beaker or something. You will not catch me in softcrit or hardcrit again, only dead or alive.
> 
> There's huge overlap in the recipe too so it's not even hard to make, literally balanced by "do you have chem access" and "is the power on and you haven't churned out 200u of this chem yet" if that.

also if someone jabs you with an epi pen while you have penthrite in your system, you literally explode, death is the result.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 5]
```diff
@@ -431,21 +431,60 @@
...
+	if(H.health <= H.crit_threshold) //certain death above this threshold
+		REMOVE_TRAIT(H, TRAIT_STABLEHEART, type) //we have to remove the stable heart before we give him heart attack
+		to_chat(H,"<span class='danger'>You feel something rupturing inside your chest!")
+		H.emote("scream")
+		H.set_heartattack(TRUE)
+		volume = 0
 	. = ..()
-	M.adjustOrganLoss(ORGAN_SLOT_STOMACH,0.5 * REM)
 
 /datum/reagent/medicine/C2/penthrite/on_mob_end_metabolize(mob/living/M)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:475]: time to spam buckling and unbuckling myself from a stasis bed to make myself be considered to be in hard crit all of the time

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 6]
```diff
@@ -431,21 +431,60 @@
...
+
+
+		if(prob(33))
+			to_chat(H,"<span class='danger'>Your body is trying to give up, but your heart is still beating!")
+
+	if(H.health <= H.crit_threshold) //certain death above this threshold
+		REMOVE_TRAIT(H, TRAIT_STABLEHEART, type) //we have to remove the stable heart before we give him heart attack
+		to_chat(H,"<span class='danger'>You feel something rupturing inside your chest!")
+		H.emote("scream")
+		H.set_heartattack(TRUE)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:471]: imagine needing a heart

this post brought to you by plasmaman gang

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 5 above)
Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:475]: relevant: https://github.com/tgstation/tgstation/issues/48904

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 7]
```diff
@@ -115,11 +115,15 @@
 	required_temp = 550 + rand(-49,49)
 
 
-/datum/chemical_reaction/reagent_explosion/penthrite_explosion
-	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/phenol = 1, /datum/reagent/acetone_oxide = 1)
-	required_temp = 315
+/datum/chemical_reaction/reagent_explosion/penthrite_explosion_epinephrine
+	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/medicine/epinephrine = 1)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm:119]: meth explosion 2, electric boogaloo

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 8]
```diff
@@ -431,21 +431,60 @@
...
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
+	REMOVE_TRAIT(M, TRAIT_NOHARDCRIT,type)
+	REMOVE_TRAIT(M, TRAIT_NOSOFTCRIT,type)
 	. = ..()
 
+/datum/reagent/medicine/C2/penthrite/overdose_process(mob/living/carbon/human/H)
+	H.adjustStaminaLoss(10)
+	H.adjustOrganLoss(ORGAN_SLOT_HEART,10)
+	H.set_heartattack(TRUE)
+	H.reagents?.chem_temp +=(-10*REM) // your heart stopped pumping. You are getting colder dear friend
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:486]: remove this temperature alteration part of the OD (or make it a part of heart attack code, but I think that it'd still be out of place there)

File: code/datums/mood_events/drug_events.dm [hunk 9]
```diff
@@ -78,3 +78,12 @@
 	description = "<span class='nicegreen'>Eh ah AAAAH! HA HA HA HA HAA! Uuuh.</span>\n"
 	mood_change = 6
 	timeout = 3 MINUTES
+
+/datum/mood_event/ldx_pain
```
Comment (ATH1909) [inline code/datums/mood_events/drug_events.dm:82]: these moodlets aren't being used for anything, why are they here?

File: code/modules/reagents/chemistry/recipes/cat2_medicines.dm [hunk 10]
```diff
@@ -61,5 +61,5 @@
 	required_reagents = list(/datum/reagent/sulfur = 1, /datum/reagent/fluorine = 1, /datum/reagent/toxin = 1, /datum/reagent/nitrous_oxide = 2)
 
 /datum/chemical_reaction/penthrite
-	results = list(/datum/reagent/medicine/C2/penthrite = 4)
-	required_reagents = list(/datum/reagent/pentaerythritol = 4, /datum/reagent/acetone = 1,  /datum/reagent/toxin/acid/nitracid = 1)
+	results = list(/datum/reagent/medicine/C2/penthrite = 3)
+	required_reagents = list(/datum/reagent/pentaerythritol = 1, /datum/reagent/acetone = 1,  /datum/reagent/toxin/acid/nitracid = 1 , /datum/reagent/ammonia = 1 , /datum/reagent/medicine/atropine = 1)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/recipes/cat2_medicines.dm:65]: you sure that the penthrite made from this won't react with the atropine that's being used to make it?

Comment (EdgeLordExe) [reply code/modules/reagents/chemistry/recipes/cat2_medicines.dm:65]: oh wait fuck you are right

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 7 above)
Comment (ATH1909) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm:119]: except this is even stronger, because it can react in your body

make two smoke grenades that are synced up to the same signaler signal. put epinephrine in one grenade and penthrite in the other. trigger them both at once, then watch as everyone nearby you who doesn't have their internals on instantly explodes

or just stick penthrite in a hypospray, then click someone once with a penthrite hypospray and once with an epinephrine pen to cause them to explode

**Comment by ATH1909** [comment]
plasmaman (or that one bioware surgery that makes you not need a heart+full augs to stop the cold damage)+memento mori+500u aranesp+500u penthrite=not caring about any damage type but brain and stamina (you'll take tox damage over time from the aranesp, but hey, who cares, you can't die/lose consciousness from damage anyway!)

actually, do you even die from brain damage if you have memento mori?

**Comment by trollbreeder** [comment]
> plasmaman (or that one bioware surgery that makes you not need a heart+full augs to stop the cold damage)+memento mori+500u aranesp+500u penthrite+stomach removal=not caring about any damage type but brain (actually, do you even die from brain damage if you have memento mori?) and stamina
> 
> perfectly balanced, as all things should be

that's called powergaming, ath

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 7 above)
Comment (EdgeLordExe) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm:119]: you discovered rdx + teslium incredibly late

**Comment by itseasytosee** [comment]
Alright I like this chem but its long past the point of medicine at this point

**Comment by ExcessiveUseOfCobblestone** [comment]
> you will have to have a heart of steel to sustain yourself on this med

no i don't because stableheart lets me live without a heart?

**Comment by ATH1909** [comment]
> > you will have to have a heart of steel to sustain yourself on this med
> 
> no i don't because stableheart lets me live without a heart?

It gets removed just before the chem gives you a heart attack.

**Comment by EdgeLordExe** [comment]
> > you will have to have a heart of steel to sustain yourself on this med
> 
> no i don't because stableheart lets me live without a heart?

yeah as long penthrite is in your system, the second it stops metabolizing you die (if you are above the crit threshold).

File: code/datums/mood_events/drug_events.dm (hunk 9 above)
Comment (trollbreeder) [reply code/datums/mood_events/drug_events.dm]: Remove these, as LDX is no longer part of this PR.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 11]
```diff
@@ -431,21 +431,60 @@
...
+		H.Jitter(rand(0,2))
+		H.Dizzy(rand(0,2))
+
+
+		if(prob(33))
+			to_chat(H,"<span class='danger'>Your body is trying to give up, but your heart is still beating!")
+
+	if(H.health <= H.crit_threshold) //certain death above this threshold
+		REMOVE_TRAIT(H, TRAIT_STABLEHEART, type) //we have to remove the stable heart before we give him heart attack
+		to_chat(H,"<span class='danger'>You feel something rupturing inside your chest!")
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:469]: Missing closing span
```suggestion
		to_chat(H,"<span class='danger'>You feel something rupturing inside your chest!</span>")
```

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 12]
```diff
@@ -431,21 +431,60 @@
...
+		H.losebreath = 0
+
+		H.adjustOrganLoss(ORGAN_SLOT_HEART,max(1,volume/10)) // your heart is barely keeping up!
+
+		H.Jitter(rand(0,2))
+		H.Dizzy(rand(0,2))
+
+
+		if(prob(33))
+			to_chat(H,"<span class='danger'>Your body is trying to give up, but your heart is still beating!")
```
Comment (trollbreeder) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:465]: Missing closing span
```suggestion
			to_chat(H,"<span class='danger'>Your body is trying to give up, but your heart is still beating!</span>")
```

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 7 above)
Comment (trollbreeder) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm:119]: except you need to use a hypospray for teslium, because there's no teslium medipens

**Comment by trollbreeder** [comment]
Compile errors.
```
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M.status_traits: undefined var
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M.comp_lookup: undefined var
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M.comp_lookup: undefined var
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M: undefined var
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M._SendSignal: undefined var
code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:483:error: M.status_traits: undefined var
```

**Comment by EdgeLordExe** [comment]
stable heart only prevents heart failure and damage from lacking a heart. It doesnt stop heart damage iirc

**Comment by EdgeLordExe** [comment]
@ExcessiveUseOfCobblestone i could potentially change the recipe to require the new lavaland geyser chems. that would probably balance it out. what do you think ?

**Comment by ExcessiveUseOfCobblestone** [comment]
do you guys play (not broken) chemistry, particularly chemfactory?

Assume that anything from the dispenser can be churned out to like hundreds of u (because it can).

**Comment by EdgeLordExe** [comment]
> do you guys play (not broken) chemistry, particularly chemfactory?
> 
> Assume that anything from the dispenser can be churned out to like hundreds of u (because it can).

i sometimes do and i feel like the way to balance this chem is either to make it use one or more of the geyser chems or make the recipe so complicated it would fill the whole chemfactory

**Comment by ExcessiveUseOfCobblestone** [comment]
geyser chem will be sufficient.

**Comment by ExcessiveUseOfCobblestone** [comment]
Most people in a fight aren't going to think to use epipen on their enemy in the middle of a fight, certainly not before the fight has been prolonged enough where the penth user should have been in crit by now.

**Comment by EdgeLordExe** [comment]
> Most people in a fight aren't going to think to use epipen on their enemy in the middle of a fight, certainly not before the fight has been prolonged enough where the penth user should have been in crit by now.

I mean yeah thats true, but thats the whole point of penthrite. You get to live longer, but as soon as anybody notices a single click and you are dead. Chem has clear advantages and disadvantages

**Comment by EdgeLordExe** [comment]
Penthrite now requires wittel

**Comment by ExcessiveUseOfCobblestone** [comment]
Before I merge this, have you considered the idea that someone uses penthrite as a poison? IE I shoot you with penthrite then follow up with epi (or just cryo it lol)?

**Comment by itseasytosee** [comment]
There are already 2 syringe gib-mixes. if you use an EpiPen that requires you to get up close which can hurt you as well.

**Comment by EdgeLordExe** [comment]
> Before I merge this, have you considered the idea that someone uses penthrite as a poison? IE I shoot you with penthrite then follow up with epi ~(or just cryo it lol)~?

yeah i made sure that penthrite (which is harder to make then teslium + rdx) has weaker explosion than RDX. Yeah i guess it is one more gib mix, but it is objectively weaker then the rest. It's explosivity can be weaponized, but there are easier better alternatives, therefore it is a weakness.

**Comment by ExcessiveUseOfCobblestone** [comment]
lovely
